agent/util-scripts/gold/pbench-clear-tools/test-66.txt

```diff
@@ -1,19 +1,31 @@
 +++ Running test-66 pbench-clear-tools --group=bad
-	pbench-clear-tools: invalid --group option ("bad"), directory not found: /var/tmp/pbench-test-utils/pbench/tools-v1-bad
+	pbench-clear-tools: invalid option ("bad"), directory not found: /var/tmp/pbench-test-utils/pbench/tools-v1-bad
```

Comment:
Do you need to update this line now? I see that Travis is unhappy....

Reply:
fixed